### ISSUE TITLE
Research if ambiguous link alerts in ANDI is an accessibility issue

### ISSUE THREAD

**Issue by AlexandracmONRR** [DOI-ONRR/onrr.gov-site#3323]
href has been updated, but is there another problem that has to be corrected?

<img width="1649" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBei83Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8264f9bf0f8445c111f7bf700f31f56782a22e05/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
Warning Alerts signal that screen reader inconsistencies are likely or information on the page may not be communicated consistently to all users.
https://www.ssa.gov/accessibility/andi/help/alerts.html

<img width="1492" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXY3Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--001f8bda7b8681e74d8e70dd197b4179835677d7/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
I believe the aria labels should include the unique name of each PDF, word doc, powerpoint files.
<img width="899" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM243Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c741f6b1ef72b6a13b6af9bf3b38889ae1d30bf8/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
Not a problem for other PDFs on preview site:
<img width="1172" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFA3Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--99177ae800b835d4f0537c589d0a5c5ef1629b20/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
@lpgoldstein - I want to get your take on this, here is my takeaway:

For PDFs on onrr.gov the aria label is considered the "accessible name", so when the aria label is the same as other aria labels but the href is unique, it is flagging it as an issue. The aria label should be unique if the href is unique.

This is not coming up as an issue for PDFs that do not have an aria label, because the "Inner text" is what is being read by the screen reader.

I think the solution for this is that the aria labels are changed to have the document's name, so that the aria labels are descriptive of the PDF. Also, any PDF that has the same href, should also have the same aria label.

**Comment by lpgoldstein** [comment]
@AlexandracmONRR That makes sense to me. I appreciate the time you took to look into this.

**Comment by AlexandracmONRR** [comment]
Jeff and Alex discussed and 3 issues came from this discussion:

- #3337
- #3336
- #3335
